packages/react-components/src/TxButton.tsx

```diff
@@ -96,7 +96,7 @@ function TxButton ({ accountId, className = '', extrinsic: propsExtrinsic, icon,
   return (
     <Button
       className={className}
-      icon={icon || 'check'}
+      icon={icon || 'check-to-slot'}
```

Comment:
```suggestion
      icon={icon || 'check'}
```

This is the default value.